masterlist.yaml

```
@@ -614,6 +614,10 @@ plugins:
       - 'Wey_New_Argonians.ESP'
 
 ###### Environment, Landscape & Flora ######
+  - name: 'Baan_Binif.ESP'
+    url: 'https://www.nexusmods.com/morrowind/mods/51419'
+    after:
+      - 'Bal''laku - The Lonely Towers.esp'
```

Closing forward slash on links.
Lists with a single entry can be formatted as a single line with square brackets.
And a newline can be should be used to separate files from different mods.

```suggestion
  - name: 'Baan_Binif.ESP'
    url: 'https://www.nexusmods.com/morrowind/mods/51419/'
    after: [ 'Bal''laku - The Lonely Towers.esp' ]

```